README.md

```diff
@@ -484,14 +484,16 @@ Similar to the JaCoCo `dump` command the flag `--coverage_dump=<file>` specifies
 `coverage.exec`, that should be generated after the fuzzing run. It contains a binary representation of the gathered
 coverage data in the JaCoCo format.
 
-The JaCoCo `report` command can be used to generate reports based on the coverage dump. For example the
-following command generates an HTML report in the folder `./report/` containing all classes available in `classes.jar`
-and their coverage as captured in the export `coverage.exec`.
+The JaCoCo `report` command can be used to generate reports based on the coverage dump. The internally used version
+differs slightly from the official one, and to prevent incorrect results it should be preferred.
```

Comment:
IMO we should word this more strongly as incorrect coverage results aren't much better than no coverage result. ;-)